AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -146,6 +157,7 @@ class TransactionConfigurator {
     func load(completion: @escaping (ResultResult<Void, AnyError>.t) -> Void) {
         switch transaction.transferType {
         case .nativeCryptocurrency, .dapp:
+            estimateGasPrice()
```

Comment:
Should `estimateGasPrice()` be called just within the `.nativeCryptoCurrency` and `.dapp` or before the `switch`?

Reply:
Yes, the other methods calculate it beforehand. With dapp transactions we don't get the chance

Reply:
Then maybe we should split the 2 cases? So that we don't override the the gas price that came along with `.nativeCryptocurrency`?

Reply:
@hboon done, don't even need the gas limit thing on natives